### PR TITLE
Feature/hot events

### PR DESCRIPTION
**Pull request by yazhyvyi**
Hot-Events on Home page

### CODE REVIEW

File: client/public/index.css
Comment (Merteg) [inline client/public/index.css]: Don't use `!important`. It's bad

File: client/src/components/HotEvents.jsx
Comment (Merteg) [inline client/src/components/HotEvents.jsx]: Why do you use curly braces here?

File: server/server/views/hot_event.py
Comment (Merteg) [inline server/server/views/hot_event.py]: I don't think that the hot events are a resource

File: server/server/views/hot_event.py
Comment (Merteg) [inline server/server/views/hot_event.py]: The logic of the server and client must be separate. What happens if you change materialize to something other on the client, you will change this part of the code on the back, it is bad. Better to handle count of the array on the front

File: server/server/views/hot_event.py
Comment (Merteg) [inline server/server/views/hot_event.py]: You have some problem here. What if event was Hot, but later you change this status to other?
(Record at Event History with status Hot will stay)

**Review by Merteg — CHANGES_REQUESTED** [review]

File: server/server/views/hot_event.py
Comment (yazhyvyi) [reply server/server/views/hot_event.py]: I think this feature should be implemented on moderator's pages, now you can only set status manually in database

**Comment by yazhyvyi** [comment]
Update

File: server/server/views/hot_event.py
Comment (Merteg) [reply server/server/views/hot_event.py]: Yes, set status Hot you can only manually, but if the user changes some data in this event, the event status will be "New" with comment - "Wait for review by moderator". All you need it's check the status with the LAST date. Not all event history, just record with the last date

**Review by Merteg — APPROVED** [review]

**Review by ykobuk — APPROVED** [review]

**Pull request merged**
